frb/figures/finder.py

```diff
@@ -190,7 +198,9 @@ def generate(image, wcs, title, log_stretch=False,
         
         plt.text(0.5, -0.05, 'Slit PA={} deg'.format(pa_deg), color='purple',
                  fontsize=15, ha='center', va='top', transform=ax.transAxes)
-        
+    
+    if ((slit is not None) and (flag_photu is False)):
+        print('Slit cannot be placed with photutils package')
```

Comment:
Turn this into a `raise IOError` and have it say
that the slit option requires that photutils be installed.

Reply:
@JayChittidi -- please get this one.  thanks.

Reply:
Hi X, the commit from yesterday has this change.